pkgs/by-name/as/assetripper/package.nix

```
@@ -0,0 +1,74 @@
+{
+  autoPatchelfHook,
+  buildDotnetModule,
+  dbus,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+buildDotnetModule (finalAttrs: {
+  pname = "assetripper";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "AssetRipper";
+    repo = "AssetRipper";
+    tag = finalAttrs.version;
+    hash = "sha256-ixXWbygFhvOjld+YRLIhkO3cgDNkQsbivri2pjU4rgM=";
+  };
+
+  buildInputs = [
+    dbus.lib
```

```suggestion
    dbus
```
This one's not necessary